pkg/converter/sloppy_file.go

```diff
@@ -172,6 +188,35 @@ func (sf *SloppyFile) convertCommand(cmd []interface{}) *string {
 	return &str
 }
 
+var resourceMemRegex = regexp.MustCompile(`^(\d+)([bkmgBKMG]){1}$`)
+
+func (sf *SloppyFile) convertMemoryResource(res string) (*int, error) {
+	const lowestMem float64 = 64 // lowest mem size sloppy.io supports
+	match := resourceMemRegex.FindStringSubmatch(res)
+	if len(match) == 3 {
```

Comment:
If I'm reading this correctly, any other length will end up assinging `app.Memory` with `nil`, effectively ignoring the invalid value, right? 

Since the `default` below throws an error ('convert resources: unsupported memory format'), it should also throw that here.

Reply:
yep. I will improve this.